src/components/ChatList/elements/ChannelImage.tsx

```diff
@@ -82,6 +95,18 @@ const Big: React.FC<ChannelImageMainProps> = ({type, image, style}) => {
 };
 
 const Small: React.FC<ChannelImageBadgeProps> = ({type}) => {
+  if (type === 'ANON_TOPIC') {
+    return (
+      <View style={[styles.badgeContainer, styles.containeranon_secondary]}>
+        <FastImage
+          source={CommunityIcon}
+          resizeMode={FastImage.resizeMode.contain}
+          style={[styles.badgeIcon, {top: -0.1, left: -0.06}]}
+        />
+      </View>
+    );
+  }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, a new condition has been added to handle the case when `type` is `'ANON_TOPIC'`. The logic seems correct as it returns a styled view with an image for this type. Again, consider extracting the repeated JSX into a separate component or function to avoid duplication.
```diff
+ if (type === 'ANON_TOPIC') {
+   return (
+     <View style={[styles.badgeContainer, styles.containeranon_secondary]}>
+       <FastImage
+         source={CommunityIcon}
+         resizeMode={FastImage.resizeMode.contain}
+         style={[styles.badgeIcon, {top: -0.1, left: -0.06}]}
+       />
+     </View>
+   );
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->